### ISSUE TITLE
Regarding Training configuration and dataset used for Hindi (Devanagari)language.

### ISSUE THREAD

**Issue by akashlp27** [JaidedAI/EasyOCR#895]
Hi, thanks for the great repo

I did have some queries regarding the dataset used and the training configuration used for the Hindi language. 
and can u guys give any how can I fine-tune the existing model for handwritten dataset.

Any inputs on learning rate and the data samples can help a lot..... Thanks.

**Comment by khawar-islam** [comment]
Hello, @akashlp27 Did you solve the fine-tunning problem?
